<launch>
  <!-- Sim Time -->
  <param name="use_sim_time" value="true" />

  <!-- Benchmarking Parameterizations -->
  <rosparam file="$(find aggressive-navigation)/launch/params/gazebo_world.yaml" command="load" />
  <rosparam file="$(find aggressive-navigation)/launch/params/path_planner.yaml" command="load" />
  <rosparam file="$(find aggressive-navigation)/launch/params/traversability.yaml" command="load" />

  <param name="world_param" type="string" value="gazebo_world_all" />
  <param name="path_param" type="string" value="path_planner_all" />
  <param name="trav_param" type="string" value="traversability_all" />
  <!-- <arg name="stack_launch" default="$(find aggressive-navigation)/launch -->
  <param name="stack_launch" type="string" value="$(find aggressive-navigation)/launch/include/robot_stack/robot_stack.launch.xml" />
  
  <node pkg="aggressive-navigation" type="benchmark_manager.py" name="benchmark_manager" />

  <!-- Rviz -->
  <node pkg="rviz" type="rviz" name="rviz" output="log" args="-d $(find aggressive-navigation)/rviz/benchmark.rviz" />
  
</launch>
